web/forms.py

```
@@ -1735,3 +1735,19 @@ class StudyGroupForm(forms.ModelForm):
     class Meta:
         model = StudyGroup
         fields = ["name", "description", "course", "max_members", "is_private"]
+
+
+class NFTBadgeForm(forms.ModelForm):
+    class Meta:
+        model = Achievement
+        fields = ["title", "description", "achievement_type", "badge_icon", "course"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)